### ISSUE TITLE
Implement the Aggregate Severity section

### ISSUE THREAD

**Issue by cgi1** [csaf-tools/CVRF-CSAF-Converter#39]
EPIC: #10

## CVRF 1.2

[Oasis v1.2 -> Aggregate Severity]http://docs.oasis-open.org/csaf/csaf-cvrf/v1.2/cs01/csaf-cvrf-v1.2-cs01.html#_Toc493508891)

## [Oasis implementation view CSAF v2](https://github.com/oasis-tcs/csaf/edit/master/csaf_2.0/prose/csaf-v2-editor-draft.md)

[Field Definiition 4.8 Aggregate Severity](https://github.com/oasis-tcs/csaf/blob/master/csaf_2.0/prose/csaf-v2-editor-draft.md#3212-document-property---aggregate-severity)

Simple mapping

<img width="810" alt="grafik" src="https://user-images.githubusercontent.com/18695252/150698641-25e912f7-ae1d-4568-a45c-1c60e925d05a.png">

**Comment by cgi1** [comment]
We dont have and CVRF 1.2  example by hand right now for Aggregate Severity.

There is an CVRF 1.1: [redhat_cvrf_2018.xml](https://github.com/csaf-tools/CVRF-CSAF-Converter/blob/main/examples/1.1/redhat_cvrf_2018.xml). 

However, parser is written for CVRF 1.1 as part of this PoC:

```
2022-01-23 22:30:40,738 - utils - CRITICAL - Input document not valid: Element '{http://www.icasi.org/CVRF/schema/cvrf/1.1}cvrfdoc': No matching global declaration available for the validation root. (<string>, line 0).
```

**Comment by zpevma** [comment]
can be closed after #48 is merged

**Comment by sustefil** [comment]
#48 merged with Aggregate severity section, closing here
